### PR TITLE
Webpack: additional resources

### PR DESCRIPTION
**Pull request by Amer-Alic**
## Because
This PR suggests adding an additional resource to this lesson.

## This PR
- The suggested additional resource: https://www.youtube.com/watch?v=5IG4UmULyoA

## Pull Request Requirements

- [x] I have thoroughly read and understand The Odin Project Contributing Guide
- [x] The title of this PR follows the location of change: brief description of change format, e.g. Intro to HTML and CSS lesson: Fix link text
- [x] The Because section summarizes the reason for this PR
- [x] The This PR section has a bullet point list describing the changes in this PR
- [ ] If this PR addresses an open issue, it is linked in the Issue section
- [ ] If any lesson files are included in this PR, they have been previewed with the Markdown preview tool to ensure it is formatted correctly
- [ ] If any lesson files are included in this PR, they follow the Layout Style Guide

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Can you explain the value in adding this new resource please @Amer-Alic? 5 assignments is leaning on the too many side of things.

**Comment by Amer-Alic** [comment]
> Can you explain the value in adding this new resource please @Amer-Alic? 5 assignments is leaning on the too many side of things.

If you watch the video, you will realise that with this video you go through almost everything in those 4 large blog posts (I was considering even to propose removing one of them, but tought it was too much). So yea, in nutshell I think this video is great foundation for this topic and other posts would be to go more in depths.

File: javascript/organizing_your_javascript_code/webpack.md [hunk 1]
```diff
@@ -15,10 +15,11 @@ This section contains a general overview of topics that you will learn in this l
...
-3. Finally, the first part of [this tutorial](https://webpack.js.org/guides/development/) talks about source maps, a handy way to track down which source file (index.js, a.js, b.js) an error is coming from when you use webpack to bundle them together. This is essential to debugging bundled code in your browser's DevTools. If the error comes from b.js the error will reference that file instead of the bundle. It also walks through an example of the `--watch` feature you _definitely_ should have taken note of above.
-4. You don't need to do the rest of the webpack tutorials at this time, but take a look at what's offered on the sidebar of their [guides page](https://webpack.js.org/guides/). There are several sweet features that you might want to use in future projects such as code-splitting, lazy-loading, and tree-shaking. Now that you have a handle on webpack's configuration system adding these features is as easy as using the right plugins and loaders!
+1. Go through [this tutorial](https://www.youtube.com/watch?v=5IG4UmULyoA). An excellent, concise tutorial that explains the most important things. Follow with an example in the video!
```
Comment (KevinMulhern) [inline javascript/organizing_your_javascript_code/webpack.md:18]:
```suggestion
1. Watch and follow along with [Fireship's Module Bundlers Explained video](https://www.youtube.com/watch?v=5IG4UmULyoA) to get a great overview and some hands on experience with webpack.
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]
Thanks for the explanation @tansaku. Please fill in the PR because prompt with that kind of thing in the future, letting us know the intention and why behind a PR helps us, help you with reviewing.

The video looks good for an overview and think it would be helpful too. I just have a some suggestions for wording in the assignment.

**Review by KevinMulhern — APPROVED** [review]
Thanks @Amer-Alic

**Pull request merged**